script.js

```diff
@@ -0,0 +1,144 @@
+const todoForm = document.querySelector('.inputForm');
+const todoInput = todoForm.querySelector('input');
+const todoList = document.querySelector('.toDos');
+const doneList = document.querySelector('.dones');
+
+const saveList = 'savedList';
+let savedList = [];
+
+const saveFinishedList = 'savedFinList';
+let savedFinList = [];
+
+loadList();
+
+// To get the input from the form by adding an
+// event listener for submit button
+todoForm.addEventListener('submit', function (e) {
+  e.preventDefault();
+
+  const input = this.inputText;
+  const inputValue = input.value;
+  // if empty, neglect it
+  if (inputValue != '') {
+    addTask(inputValue);
+    saveTasks(inputValue);
+  }
+  //when submitted, reset the input form
+  todoForm.reset();
+});
+
+function saveFinTasks(todo) {
+  savedFinList.push(todo);
+  localStorage.setItem(saveFinishedList, JSON.stringify(savedFinList));
+}
+
+function saveTasks(todo) {
+  savedList.push(todo);
+  localStorage.setItem(saveList, JSON.stringify(savedList));
+}
+
+function loadList() {
+  const loadedList = localStorage.getItem(saveList);
+  if (loadedList !== null) {
+    const tempList = JSON.parse(loadedList);
+
+    for (let i of tempList) {
+      addTask(i);
+      saveTasks(i);
+    }
+  }
+  const loadedFinList = localStorage.getItem(saveFinishedList);
+  if (loadedFinList !== null) {
+    const tempList = JSON.parse(loadedFinList);
+
+    for (let i of tempList) {
+      addFinTask(i);
+      saveFinTasks(i);
+    }
+  }
+}
+
+function addFinTask(task) {
+  const li = document.createElement('li');
+  const span = document.createElement('span');
```

Comment:
태그 이름을 그대로 변수 이름으로 사용한다면 다른사람이 코드를 볼 때 이 변수의 기능을 바로 알아차리지 못할 것 같다는 생각이 들어요...!
```suggestion
  const newFinishedTodoItem = document.createElement('li');
  const todoItemContent = document.createElement('span');
```